files/InfinumCodeStyle.xml

```diff
@@ -1,34 +1,6 @@
-<code_scheme name="InfinumCodeStyle">
-  <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-  <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="99" />
-  <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">
-    <value />
-  </option>
-  <option name="IMPORT_LAYOUT_TABLE">
-    <value>
-      <package name="android" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="com" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="junit" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="net" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="org" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="java" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="javax" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="" withSubpackages="true" static="false" />
-      <emptyLine />
-      <package name="" withSubpackages="true" static="true" />
-      <emptyLine />
-    </value>
-  </option>
+<code_scheme name="InfinumCodeStyle" version="173">
```

Comment:
Is this the name visible in AS when selecting code style?
Could we change it to InfinumCodeStyle 2.0 or similar?

Reply:
It is and I was considering doing 2.0, but I'm not sure how much we want to be versioning this. I thought it'd be weird to have "InfinumCodeStyle 4.5" at some point, but maybe not.

Reply:
I don't care about versioning, could be InfinumCodeStyleNew if you ask me :D

Reply:
I'd rather it was just "Infinum" tbh, it's already obvious from the XML name and where you're importing/picking it from so the "CodeStyle" suffix is redundant.